docs/releases.md

```diff
@@ -0,0 +1,28 @@
+# Releases
+
+## Supported Versions
+
+Versions are expressed as x.y.z, where x is the major version, y is the minor version, and z is the patch version,
+following Semantic Versioning terminology.
+
+We maintain release branches for the most recent two minor releases.
+
+Fixes may be backported to release branches, depending on severity, risk, and, feasibility.
```

Comment:
Are issues like https://github.com/argoproj/argo-workflows/issues/5326 for this purpose? I see maintainers have edited the issue description. Could we document that process as well? Is this a place where community members can chime in and see if there other fixes that should be backported?

Reply:
There is no formal process for this.